src/hotspot/share/compiler/compilerOracle.cpp

```diff
@@ -637,6 +637,10 @@ static void usage() {
   tty->print_cr("and 'compileonly'. There is no priority of commands. Applying (a subset of) these");
   tty->print_cr("commands to the same method results in undefined behavior.");
   tty->cr();
+  tty->print_cr("The 'exclude' command excludes methods from top-level compilations as well as");
+  tty->print_cr("from inlining whereas the 'compileonly' command only excludes methods from");
```

Comment:
```suggestion
  tty->print_cr("from inlining, whereas the 'compileonly' command only excludes methods from");
```

Reply:
Thanks for the suggestion. Updated as proposed by you.